app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -180,20 +180,20 @@ def parse_openshift_image(openshift_image)
         new_result[:tag] ||= json["tag"] if json.keys.include?("tag")
 
         docker_metadata = openshift_image[:dockerImageMetadata]
+        docker_labels = parse_identifying_attributes(docker_metadata[:Config][:Labels], 'docker_labels', "openshift")
+        labels = parse_labels(openshift_image)
         new_result.merge!(
           :architecture          => docker_metadata[:Architecture],
           :author                => docker_metadata[:Author],
           :command               => docker_metadata[:Config][:Cmd],
           :entrypoint            => docker_metadata[:Config][:Entrypoint],
           :docker_version        => docker_metadata[:DockerVersion],
-          :exposed_ports         => parse_exposed_ports(
-            docker_metadata[:Config][:ExposedPorts]),
-          :environment_variables => parse_env_variables(
-            docker_metadata[:Config][:Env]),
+          :exposed_ports         => parse_exposed_ports(docker_metadata[:Config][:ExposedPorts]),
+          :environment_variables => parse_env_variables(docker_metadata[:Config][:Env]),
           :size                  => docker_metadata[:Size],
           :labels                => parse_labels(openshift_image),
-          :docker_labels         => parse_identifying_attributes(docker_metadata[:Config][:Labels],
-                                                                 'docker_labels', "openshift"),
+          :docker_labels         => docker_labels,
+          :tags                  => map_labels('ContainerImage', labels + docker_labels)
```

Comment:
> @simon3z Im only merging them when sending them to be mapped into tags.

@zeari it's not about how many times or where. If you do it even once you still have to answer the questions above (what if they have the same key but different values? Which one should win? why?).

Is this used by the Chargeback for Images? (IIRC we were basing it on labels)

Let's remove everything that could cause issues and that it's not vital for the use-cases we identified.